docker-compose.yml

```diff
@@ -0,0 +1,26 @@
+version: '3'
+
+services:
+
+  postgres:
+    build:
+      context: .
+      dockerfile: Dockerfile.db
+    ports:
+      - "5432:5432"
+    volumes:
+      - ./pgdata:/var/lib/postgresql/data
+    env_file:
+      - .db.env.example
```

Comment:
I would set these to `.db.env`, so deployer just has to create the env file, not modify this file, to get started.